clusterloader2/pkg/measurement/util/object_store_dynamic.go

```diff
@@ -58,14 +58,14 @@ func NewDynamicObjectStore(ctx context.Context, dynamicClient dynamic.Interface,
 	}, nil
 }
 
-// ListConditions returns list of conditions for each object that was returned by lister.
-func (s *DynamicObjectStore) ListConditions() ([][]metav1.Condition, error) {
+// ListObjectsWithConditions returns list of objects with conditions for each object that was returned by lister.
+func (s *DynamicObjectStore) ListObjectsWithConditions() ([]ObjectSimplification, error) {
```

Comment:
Nit: could we rename this method to `ListObjectSimplifications`?

I find this name more explicit and, at the same time, I'd expect that `Objects` by default have `conditions` included, hence the redundancy of `WithConditions` suffix.

Reply:
ack, done